tox.ini

```diff
@@ -24,6 +24,7 @@ deps =
     pytest-travis-fold==1.3.0
     pytest-cov==2.6.0
     PyYAML==5.1.2
+    toml
```

Comment:
The problem with adding this here is that now we've lost coverage for Import Linter when toml isn't installed. It would be best if we could have separate Tox test environment to run tests when it is and isn't installed.

Happy to discuss in more detail about how best to do this.

Reply:
Can you please help here since I'm not familiar with tox